File: core.shared/Typescript/DTO/V1/CaptainWallet/participantWalletminiscule.ts
/* Auto Generated */

import { AssurePrincipal } from "./assurePrincipal";
import { Beneficiaires } from "./beneficiaires";

export interface ParticipantWalletminiscule {
    contratResponsable: string;
    type: string;
    numeroAMC: string;
    qrCode: string;
    assurePrincipal: AssurePrincipal;
    beneficiaires: Beneficiaires[];
    debutValidite: string;
    finValidite: string;
    carte_active: string;
    codeGR: string;
    assistanceReference: string;
    assistanceContact: string;
    complementGR: string;
    statut: string;
}

File: core.shared/Net/DTO/V1/CaptainWallet/ParticipantWalletminiscule.cs
namespace Core.Shared.Net.DTO.V1.CaptainWallet
{
    public class ParticipantWalletminiscule
    {
        public string contratResponsable { get; set; }
        public string type { get; set; }
        public string numeroAMC { get; set; }
        public string qrCode { get; set; }
        public AssurePrincipal assurePrincipal { get; set; }
        public IList<Beneficiaires> beneficiaires { get; set; }
        public string debutValidite { get; set; }
        public string finValidite { get; set; }
        public string carte_active { get; set; }
        public string codeGR { get; set; }
        public string assistanceReference { get; set; }
        public string assistanceContact { get; set; }
        public string complementGR { get; set; }
        public string statut { get; set; }
    }
}
